packages/lit-dev-content/site/tutorials/content/carousel/06.md

```diff
@@ -0,0 +1,66 @@
+The slots need to be updated to correctly render the previous and currently
+selected items.
+
+Add this code above the `selected` property accessor:
+
+{% switchable-sample %}
+
+```ts
```

Comment:
Nit: Need to add `query` import either here or in the before code.

Reply:
Fixed.